server/db/database.js

```diff
@@ -170,7 +170,7 @@ export function queryPromise(sql, values = []) {
   return new Promise(function (resolve, reject) {
     connection.query(sql, values, function (err, res) {
       if (err) {
-        reject(err);
+        reject(addErrorTitle(err, "MySql error"));
```

Comment:
Second thoughts: should internal (e.g. SQL) errors be sent back to the end user? It could increase the area of attack.
We may want to log them instead, server-side

Reply:
This is what is done, this function rejects an error with a title for clarity in the logs, but the `queryPromise` errors are still hidden from the user.

Reply:
Sorry, I forgot what `addErrorTitle()` do :sweat_smile: